src/main/java/com/uber/cadence/worker/ShadowingOptions.java

```diff
@@ -162,7 +163,14 @@ public ShadowingOptions build() {
         throw new IllegalArgumentException(
             "workflow types, status and start time filter can't be specified when workflow query is specified");
       }
-
+      if (workflowQuery.isEmpty()) {
```

Comment:
I probably missed this in the previous diff. Would you mind provide some context behind checking for OPEN status on L161-162?

Reply:
The default workflow status is OPEN. Here it checks if both query  and filter are setup

Reply:
oh, I see. I think I missed this part in go client's implementation. Right now in go, if no status is specified it will query all workflows.

Reply:
I see. I set the default to be OPEN workflows